### ISSUE TITLE
result-image-format doesn't work

### ISSUE THREAD

**Issue by psymatic** [CrackerakiUA/ui-cropper#33]
Option "result-image-format" doesn't work. You can set this option to "image/jpeg" and even choose original image in jpeg format for cropping, but while you upload cropped image to server you can see that Content-Type is image/png, not image/jpeg!

**Comment by jgribonvald** [comment]
The version 1.0.4 solve this problem, but when using npm module doesn't apply the fix into compile ui-cropper.js files, you have to apply the fix on #27

**Comment by rickerd** [comment]
@jgribonvald does it mean that 1.0.4 does not have a the complete fix? Does a new version 1.0.5 needs to be made?
Please reply so i can make it correct.

**Comment by jgribonvald** [comment]
Yes you need to generate a complete version change, and yes you need to make a new version for that.

**Comment by rickerd** [comment]
Got it! 
I'll talk about a new version.

**Comment by geodeath** [comment]
Hi all!

We are using this through bower and it seems that the fix mentioned above does not solve the problem. Anyhow, i am not sure how to even solve the problem 'properly', patching the minified compiled file is not a solution as other devs will run bower install and get the 1.0.4 version as-is.

Any suggestions on how to fix for good? 

thanks!

**Comment by rickerd** [comment]
I you make a fix without the minified version you can push your changes in a PR towards us so everybody will have a new working version?

I'm not sure what you mean.

**Comment by geodeath** [comment]
What i mean is that by reading through these comments, it seems to be fixed, but not released. The posts above seem to suggest that the source of 1.0.4 is fine, but that the compiled version is not. I can confirm this, as they do look different.

However, i did try the fix above and it still does not work as expected. What i am asking is:

1. Did anyone fix this? If so, what exactly did you have to patch? Changing the "=?" to "@" does not fix this problem, at least applying it to the minified version.
2. If this was actually fixed in the source (as suggested above), how do i compile it to the minified version and then make a new version for the benefit of everybody?

Apologies for being a noob, i want to fix and help, but have no clue and what i tried did not work :)

Feel free to inform and suggest what i should do, even if it is to read up something. It is just that the information on this thread is confusing and it does not resolve the problem.

**Comment by rickerd** [comment]
I don't know also. I'm not developing on this project as much and don't follow all the issues that are being made.
Open Source to me is like if you need something or something is broken, fix it or implement it yourself.

I think that the 1.0.4 version is not compiled correct.

Can you;
- clone the repo instead of "downloading" it via bower.
- copy the cloned repo into the bower_components folder where the "old" version used to be of uicropper
- npm install everything
- gulp and gulp dist/build
- run your own build scripts of the project where the cropper is being used and check if everything works.

I don't know why it's not working but somewhere in the source the image is not changed to jpg and kept on png.
Why that is... i have no clue.

**Comment by geodeath** [comment]
Thanks man, appreciated. hopefully i will be able to fix it :)
